src/joinRates/__test__/JoinRates.mock.ts

```diff
@@ -0,0 +1,319 @@
+export const mockData1 = {
```

Comment:
Why do we need this? Should we have just the one json sample file and be good for tests and stories?

Reply:
We have some cases that are not covered by mock from stories.
If we will use only mocks from stories we need to add different cases to them